### ISSUE TITLE
LLVM Error

### ISSUE THREAD

**Issue by gareins** [blackbeam/rust-mysql-simple#13]
Compiling mysql v0.3.1
LLVM ERROR: expected relocatable expression
Could not compile `mysql`.

This one is new as of when I updated Rust 2 hours ago. Should this be reported upstream to the Rust team?

**Comment by blackbeam** [comment]
I tried hard but could not reproduce. Which version of rust did you use?

**Comment by gareins** [comment]
I updated again this morning, nothing changed
Version: rustc 1.0.0-nightly (2b01a37ec 2015-02-21) (built 2015-02-22)

```

flash@flash-VirtualBox:~/develop/Zalka$ uname -a
Linux flash-VirtualBox 3.13.0-45-generic #74-Ubuntu SMP Tue Jan 13 19:37:48 UTC 2015 i686 i686 i686 GNU/Linux
flash@flash-VirtualBox:~/develop/Zalka$ rustc --version
rustc 1.0.0-nightly (2b01a37ec 2015-02-21) (built 2015-02-22)
flash@flash-VirtualBox:~/develop/Zalka$ cargo -V
cargo 0.0.1-pre-nightly (43755c0 2015-02-19) (built 2015-02-21)
flash@flash-VirtualBox:~/develop/Zalka$ cargo build --verbose
       Fresh pkg-config v0.2.2
       Fresh libc v0.1.2
       Fresh fern v0.1.12
       Fresh rustc-serialize v0.2.15
       Fresh gcc v0.2.1
       Fresh regex v0.1.15
       Fresh bitflags v0.1.1
       Fresh gcc v0.1.7
       Fresh lazy_static v0.1.7
       Fresh fern_macros v0.1.6
       Fresh regex_macros v0.1.8
       Fresh openssl-sys v0.4.3
       Fresh time v0.1.17
       Fresh openssl v0.4.3
       Fresh chrono v0.2.1
   Compiling mysql v0.3.1
     Running `rustc /home/flash/.cargo/registry/src/github.com-1285ae84e5963aae/mysql-0.3.1/src/lib.rs --crate-name mysql --crate-type lib -g --cfg feature="default" --cfg feature="ssl" --cfg feature="openssl" -C metadata=1dd4b5094940a1c2 -C extra-filename=-1dd4b5094940a1c2 --out-dir /home/flash/develop/Zalka/target/deps --emit=dep-info,link -L dependency=/home/flash/develop/Zalka/target/deps -L dependency=/home/flash/develop/Zalka/target/deps --extern bitflags=/home/flash/develop/Zalka/target/deps/libbitflags-fef903dcb95d8cb2.rlib --extern openssl=/home/flash/develop/Zalka/target/deps/libopenssl-34d5d63c9072d2be.rlib --extern regex_macros=/home/flash/develop/Zalka/target/deps/libregex_macros-3de0438ced8fca19.so --extern regex=/home/flash/develop/Zalka/target/deps/libregex-9edc303929b59965.rlib --extern lazy_static=/home/flash/develop/Zalka/target/deps/liblazy_static-62d7487473abda8f.rlib --extern time=/home/flash/develop/Zalka/target/deps/libtime-e9fcf4d7331b5e65.rlib -Awarnings -L native=/usr/lib/i386-linux-gnu -L native=/home/flash/develop/Zalka/target/build/openssl-sys-62bda7d2203cce95/out -L native=/home/flash/develop/Zalka/target/build/time-e9fcf4d7331b5e65/out`
LLVM ERROR: expected relocatable expression
Could not compile `mysql`.

Caused by:
  Process didn't exit successfully: `rustc /home/flash/.cargo/registry/src/github.com-1285ae84e5963aae/mysql-0.3.1/src/lib.rs --crate-name mysql --crate-type lib -g --cfg feature="default" --cfg feature="ssl" --cfg feature="openssl" -C metadata=1dd4b5094940a1c2 -C extra-filename=-1dd4b5094940a1c2 --out-dir /home/flash/develop/Zalka/target/deps --emit=dep-info,link -L dependency=/home/flash/develop/Zalka/target/deps -L dependency=/home/flash/develop/Zalka/target/deps --extern bitflags=/home/flash/develop/Zalka/target/deps/libbitflags-fef903dcb95d8cb2.rlib --extern openssl=/home/flash/develop/Zalka/target/deps/libopenssl-34d5d63c9072d2be.rlib --extern regex_macros=/home/flash/develop/Zalka/target/deps/libregex_macros-3de0438ced8fca19.so --extern regex=/home/flash/develop/Zalka/target/deps/libregex-9edc303929b59965.rlib --extern lazy_static=/home/flash/develop/Zalka/target/deps/liblazy_static-62d7487473abda8f.rlib --extern time=/home/flash/develop/Zalka/target/deps/libtime-e9fcf4d7331b5e65.rlib -Awarnings -L native=/usr/lib/i386-linux-gnu -L native=/home/flash/develop/Zalka/target/build/openssl-sys-62bda7d2203cce95/out -L native=/home/flash/develop/Zalka/target/build/time-e9fcf4d7331b5e65/out` (status=1)

```

**Comment by gareins** [comment]
Running with --emit=llvm-ir got me this: https://mega.co.nz/#!1BIACQYa!bfbFQOfyo-6enEN--FRmLqRDB2WoAPwhSCEBqFRtIKY

Updated mysql from 5.5 to 5.6, still LLVM error.

**Comment by gareins** [comment]
Thanks!

**Comment by blackbeam** [comment]
Published as v0.3.2.
